tests/test_table.py

```diff
@@ -341,8 +341,8 @@ def test_update_expressions(TestModel, TestModel_entries, dynamo_local):
 
     two.update(things=['foo'])
     assert two.things == ['foo']
-    two.update(things__append=[1])
-    assert two.things == ['foo', 1]
+    two.update(things__append=['bar'])
+    assert two.things == ['foo', 'bar']
```

Comment:
Let's change the test to use the [`DynamoTimestampType`](https://github.com/NerdWalletOSS/dynamorm/blob/8a7e43e/tests/conftest.py#L96) so that we're actually testing something that uses different value for native & primitive values.

Reply:
Done